ern/graphql.go

```diff
@@ -0,0 +1,552 @@
+// This file is part of the go-meta library.
+//
+// Copyright (C) 2017 JAAK MUSIC LTD
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+// If you have any questions please contact yo@jaak.io
+
+package ern
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-ipld-format"
+	"github.com/meta-network/go-meta"
+)
+
+// General purpose GraphQL resolver function,
+// retrieves data from a META store and SQLite3 index
+type Resolver struct {
+	db    *sql.DB
+	store *meta.Store
+}
+
+// Returns a new resolver for returning data from the given
+// META store and SQLite3 index
+func NewResolver(db *sql.DB, store *meta.Store) *Resolver {
```

Comment:
Comment this exported func

// NewResolver ...

Reply:
Thanks